src/sentry/api/endpoints/group_details.py

```diff
@@ -206,24 +199,64 @@ def get(self, request, group):
         if last_release:
             last_release = self._get_release_info(request, group, last_release)
 
-        try:
-            environment_id = self._get_environment_id_from_request(
-                request, group.project.organization_id)
-        except Environment.DoesNotExist:
-            get_range = lambda model, keys, start, end, **kwargs: \
-                {k: tsdb.make_series(0, start, end) for k in keys}
-            tags = []
-            user_reports = UserReport.objects.none()
-
-        else:
+        # TODO(jess): This is a little gross right now, but I think we should be able
```

Comment:
yeah so what i was thinking is that it probably makes sense to still have a conditional like you do here https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/group_events.py#L68 which respects a global option too (so if we decide to flip snuba off for some reason, that is respected for the requests from the old version of issue details). @alex-hofsteede is there an existing one I could piggy back off of for now?

Reply:
um, yeah depends on your specific use case. `snuba.events-queries.enabled` is probably going to stay turned off for a while until I can land https://github.com/getsentry/sentry/pull/11071 but feel free to register a new one that switches this on/off in particular

Reply:
is this too gross? https://github.com/getsentry/sentry/pull/11781/files#diff-165d90922c56f54ff6874db9c9eb1489R204

Reply:
I guess I'm not quite sure what the state of everything is supposed to be. IIUC this endpoint is getting hit from new under-development features that need the multi-env capability of snuba, but its also being hit from the existing issue details page. 

Are we confident enough in the snuba version to make it live for all use cases? or do we want to keep the old implementation for the old issue details page for now?

Reply:
Good question. The differences are pretty minimal and only affect the various "seen" stats. If we make it based on the tagstore setting, the only serializer differences are that the non-snuba version uses `tagstore.get_group_list_tag_value` while the snuba one uses `tagstore.get_group_seen_values_for_environments` and also the `GroupEnvironment.first_seen` column. So, I think if anything, it would probably be more accurate because of the GroupEnvironment use. That said, this is one of our most important endpoints, so maybe the best call is to rely entirely on the query param to decide which serializer to use so the old version is unaffected for now. That way we can sanity check it for a few days and then cut over.

Reply:
Sounds good to me